babel.esm.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: './babel.build.config.js',
+  presets: [
+    // Dont transform modules for the esm build
+    ['@babel/preset-env', { modules: false }],
+  ],
+};
```

Comment:
Just curious, does `esm` build not transpile typescript and jsx?

Reply:
It does. The module syntax `import` is not transformed.